File: KnowledgeSharingApi/KnowledgeSharingApi.Domain/Models/Entities/Tables/PostEditHistory.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace KnowledgeSharingApi.Domains.Models.Entities.Tables
{
    [Table("ProfileEditHistory")]
    public class PostEditHistory : Entity
    {
        public Guid PostEditHistoryId { get; set; }
        public Guid PostId { get; set; }
        public string? Title { get; set; }
        public string? Abstract { get; set; }
        public string? Thumbnail { get; set; }
        public string? Content { get; set; }

        protected override PostEditHistory Init()
        {
            return new PostEditHistory();
        }
    }
}

File: KnowledgeSharingVue/knowledge-sharing-vue/src/js/models/entities/post-edit-history.js

import Entity from './entity';
class PostEditHistory extends Entity{
    constructor() {
        super();
        this.postEditHistoryId = null;
        this.postId = null;
        this.title = null;
        this.abstract = null;
        this.thumbnail = null;
        this.content = null;
    }

    init() {
        return new PostEditHistory();
    }
}

export default PostEditHistory;